Makefile

```diff
@@ -75,32 +99,33 @@ docs-user: clean-docs
 docs: docs-user docs-developer
 
 release:
-	ls -l dist/
-	read "\nDo you want to upload everything in dist/*?\n\n CTRL+C to exit."
+	@ls -l dist/
+	@read "\nDo you want to upload everything in dist/*?\n\n CTRL+C to exit."
 	twine upload -s dist/*
 
 staticdeps:
 	rm -r kolibri/dist/* || true # remove everything
 	git checkout -- kolibri/dist # restore __init__.py
-	pip install -t kolibri/dist -r $(REQUIREMENTS)
-	python install_cexts.py --file $(REQUIREMENTS_CEXT) # pip install c extensions
-	pip install -t kolibri/dist -r $(REQUIREMENTS_CEXT_NOARCH) --no-deps
+	pip install -t kolibri/dist -r "requirements.txt"
+	python install_cexts.py --file "requirements/cext.txt" # pip install c extensions
+	pip install -t kolibri/dist -r "requirements/cext_noarch.txt" --no-deps
 	rm -r kolibri/dist/*.dist-info  # pip installs from PyPI will complain if we have more than one dist-info directory.
 
 writeversion:
 	python -c "import kolibri; print(kolibri.__version__)" > kolibri/VERSION
+	@echo ""
+	@echo -n "Current version is now "
```

Comment:
When doing `make writeversion`, the result I got is
```
-n Current version is now 
0.8.0.dev020180124211410-git
```

I assume what we want here is something like 
```
Current version is now 0.8.0.dev020180124211410-git
```
I guess having `-n` here is to not print the trailing newline character? I found that in [Makefile](https://www.gnu.org/software/make/manual/html_node/Options-Summary.html#Options-Summary), the option `-n` stands for "don't execute", which overwrites `echo`'s option `-n`. That's why we are getting `-n Current version is now` instead of `Current version is now`. One way I think might work is changing the line to `@echo "Current version is now \c". But there may be a better way to handle this :)

Reply:
Well actually this didn't happen on my system, but will still use a different approach, just to avoid spreading a bad pattern.